CHANGELOG.md

```diff
@@ -1,9 +1,38 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+This project adheres to [Semantic Versioning](http://semver.org/)
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## 1.2.0 2017-05-13
+## [Unreleased]
+
+## [2.0.0] - 2018-02-07
```

Comment:
Generally you don't version it in the PR but this is a special case so it's fine.